lib/generators/declare_schema/migration/migrator.rb

```diff
@@ -110,6 +112,11 @@ def fix_native_types(types)
           def native_types
             @native_types ||= fix_native_types(connection.native_database_types)
           end
+
+          def after_load_rails_models(&blk)
+            block_given? or raise ArgumentError, 'A block is required when setting the after_load_rails_models callback'
```

Comment:
Nit: Given that you have `&blk` argument, I think it's slightly more consistent to use
```
blk or raise ...
```
And why not call it `block`?

Reply:
Good point. Thank you. I slipped into old ruby habits with the naming of `blk` instead of `block`